### PR TITLE
Uniformize image/sound loading

### PR DESCRIPTION
**Pull request by mgodlewski**

### CODE REVIEW

File: bambam.py [hunk 1]
```diff
@@ -69,15 +69,15 @@ def play(self): pass
     return sound
 
 
-# Loads a list of sounds
-def load_sounds(lst):
+# Runs load_function on elements of lst unless they are blacklisted.
```
Comment (porridge) [inline bambam.py]: Please move this line to a docstring below the def.

File: bambam.py [hunk 2]
```diff
@@ -69,15 +69,15 @@ def play(self): pass
...
+def load_items(lst, blacklist, load_function):
     result = []
     global args
```
Comment (porridge) [inline bambam.py]: Remove this line, not needed anymore.

**Comment by porridge** [comment]
Done reviewing.

File: bambam.py (hunk 1 above)
Comment (mgodlewski) [inline bambam.py]: The doctring convention is not used in the rest of the code.
I porpose to migrate to doctring convention for all methods in a dedicated commit, in another pullrequest.
What is your opinion ?

File: bambam.py (hunk 2 above)
Comment (mgodlewski) [inline bambam.py]: I do not understand which line you want I remove.

Comment (porridge) [inline bambam.py]: ```global args```, it's not needed anymore, right?

File: bambam.py (hunk 1 above)
Comment (porridge) [inline bambam.py]: I'm not sure it's worth investing that time, up to you.

**Comment by porridge** [comment]
Looks good, please squash before I merge.
Also, for the future, please ping me when you push another commit, gihub does not notify about that.

**Pull request merged**